www_src/pages/project/project.jsx

```diff
@@ -278,6 +306,10 @@ var Project = React.createClass({
       state.camera = this.cartesian.getFocusTransform({x: 0, y: 0}, this.state.zoom);
     }
     this.setState(state);
+
+    if (this.state.params.mode === 'linkDestination') {
+      this.highlightPage('408', 'source'); // TEMP / TODO - Hardcoded. Should come from passed value.
```

Comment:
Note this hardcoded page ID...works as of now, but if the project loses this page you might need to set another ID to see this working...